#! /bin/sh -e

# DP: Fix gpc build error for bootstrap target

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        #cd ${dir}gcc && autoconf
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        #rm ${dir}gcc/configure
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- gcc/p/Make-lang.in.bb	Mon May 17 12:16:51 2004
+++ gcc/p/Make-lang.in	Mon May 17 12:17:21 2004
@@ -1199,7 +1199,8 @@
 
 RTSSTAGESTUFF=p/rts/*.o p/rts/*.lo p/rts/*.gpi p/rts/*.gpd \
   p/rts/config.cache p/rts/config.log p/rts/config.status p/rts/Makefile \
-  p/rts/rts-config.h p/rts/rts-config.inc p/rts/needed-options
+  p/rts/rts-config.h p/rts/rts-config.inc p/rts/needed-options \
+  p/rts/stamp-error-gpi
 
 pascal.stage1:
 	$(srcdir)/p/script/mkdir-p stage1/p/rts


-- 
                              Waldek Hebisch
hebisch@math.uni.wroc.pl 
